### PR TITLE
Fix hamburger menu button color on xda-developers.com

### PR DESCRIPTION
**Pull request by KyleGospo**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3537,6 +3537,18 @@ INVERT
 
 ================================
 
+xda-developers.com
+
+CSS
+.hb-trigger-inner, .hb-trigger-inner::before, .hb-trigger-inner::after {
+    background-color: rgb(171, 171, 171) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3544]: `background-color: ${rgb(48, 51, 53)} !important`
Try to use ${}

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:3544]: For the rest a clean CSS Fix

**Comment by KyleGospo** [comment]
Change made, thanks for the tip!

**Comment by Gusted** [comment]
Thank you for contributing :D

**Pull request merged**